osu.Game/Screens/Select/Details/AdvancedStats.cs

```diff
@@ -223,6 +221,7 @@ public LocalisableString Title
                     bar.Length = value.baseValue / maxValue;
 
                     valueText.Text = (value.adjustedValue ?? value.baseValue).ToString(forceDecimalPlaces ? "0.00" : "0.##");
+                    if (value.isRateAdjusted) valueText.Text += "*";
```

Comment:
Ignoring the code quality of this PR (I am not ready to even attempt to figure it out), this method of display is ambiguous. A user would have little idea what this means.